### DISCUSSION TITLE
How do I import modules from github if they are using `require` internally?

### DISCUSSION THREAD

**Discussion by shivam-tripathi** [denoland/deno#18628] [Q&A]
Thanks to the amazing work by the team, we can now seamlessly import modules npm with the `npm:` specifier. 

I was trying to import some libraries directly from github but it errors out with 
```error: Uncaught ReferenceError: require is not defined```.

Is there is an easy way to import directly from github packages with commonjs imports? 
Really grateful to the team for all the amazing work!

**Comment by ngdangtu-vn** [comment]
On Deno 1.44 (arrived in 2024) supports this feature: https://deno.com/blog/v1.44 (I leave the link here in case sb passing by and need an answer)
